src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Headers.cs

```diff
@@ -94,6 +97,65 @@ public async Task SendAsync_SpecialCharacterHeader_Success()
             });
         }
 
+        [Fact]
+        public async Task GetAsync_MissingExpires_ReturnNull()
+        {
+             await LoopbackServerFactory.CreateClientAndServerAsync(async uri =>
+             {
+                using (var client = CreateHttpClient())
+                {
+                    HttpResponseMessage response = await client.GetAsync(uri);
+                    Assert.Null(response.Content.Headers.Expires);
+                }
+            },
+            async server =>
+            {
+                await server.HandleRequestAsync(HttpStatusCode.OK);
+            });
+        }
+
+        [Theory]
+        [InlineData("Thu, 01 Dec 1994 16:00:00 GMT", true)]
+        [InlineData("-1", false)]
+        [InlineData("0", false)]
+        public async Task SendAsync_Expires_Success(string value, bool isValid)
+        {
+            await LoopbackServerFactory.CreateClientAndServerAsync(async uri =>
+            {
+                using (var client = CreateHttpClient())
+                {
+                    var message = new HttpRequestMessage(HttpMethod.Get, uri);
+                    HttpResponseMessage response = await client.SendAsync(message);
+                    Assert.NotNull(response.Content.Headers.Expires);
+                    if (!isValid)
+                    {
+                        // Invalid date should be converted to MinValue so everything is expired.
+                        Assert.Equal(DateTimeOffset.MinValue, response.Content.Headers.Expires);
+                    }
```

Comment:
When `isValid` is true, shouldn't we validate that Expires returns the expected value?